lib/developer_portal/lib/liquid/tags/active_docs.rb

```diff
@@ -16,19 +16,19 @@ def initialize(tag_name, params, tokens)
         super
         if params =~ Syntax
           @version  = $1.strip
-          @services = $2.nil? ? [] : $2.split(",").map(&:strip)
+          @system_names = $2.nil? ? [] : $2.split(",").map(&:strip)
         end
       end
 
       def render context
-        if @services.empty? && @version > "1.2"
-          @services =
-            context.registers[:controller].send(:site_account)
-            .api_docs_services.published
-            .select{|e| e.specification.swagger_2_0?}.map(&:system_name)
+        provider_account = context.registers[:controller].send(:site_account)
```

Comment:
I know this is not your fault, but if `site_account` is private, I feel that we are doing something not really cool... 😞 Maybe it should be public and call it directly in a normal way... but I am not sure. WDYT @josemigallas @hallelujah @guicassolato @macejmic ?